code/game/jobs/job/marines.dm

```diff
@@ -72,12 +72,12 @@ Your squaddies will look to you when it comes to construction in the field of ba
 
 
 /datum/job/marine/medic
-	title = "Squad Medic"
+	title = "Squad Corpsman"
 	comm_title = "Med"
-	paygrade = "E4"
+	paygrade = "E3"
```

Comment:
What are we doing here? HN (Hospital Corpsman)? PO (Petty Officer)?

Reply:
Was just gonna stick with LCPL like you said, referencing the person you conversed with about how they would integrate ranks. It could be changed once/if the database promotion system is made.